fastlane/lib/fastlane/helper/adb_helper.rb

```diff
@@ -39,7 +39,7 @@ def trigger(command: nil, serial: nil)
         Action.sh(command)
       end
 
-      def device_avalaible?(serial)
+      def device_available?(serial)
```

Comment:
This misspelling is okay because `device_available?` is correct below 👇 This is a misspelling that was cause earlier and is now acting as an alias

Reply:
Ah, I missed this deprecate message. Thanks.

Reply:
> Revert accidentally typofix

This might be my favorite commit message of all time